DEPLOYMENT.md

```diff
@@ -17,7 +17,7 @@ docker run -it --rm --entrypoint python3 athenian/api -m athenian.api.models.sta
 The server requires:
 
 - (optional) `SENTRY_KEY` and `SENTRY_PROJECT` environment variables to enable error logging.
-- (optional) `AUTH0_DOMAIN` and `AUTH0_AUDIENCE` environment variables to enable authorization.
+- `AUTH0_DOMAIN` and `AUTH0_AUDIENCE` environment variables to enable authorization.
```

Comment:
It is impossible to have CRUD without auth. Instead of having two different server execution modes, I hardcoded the user for now.

Reply:
> Instead of having two different server execution modes, I hardcoded the user for now.

This totally makes sense IMO for now. We'll then need to think about how to tackle having a public dashboard for dogfooding using the same SaaS instance.